src/server/announcements.ts

```diff
@@ -13,20 +14,38 @@ export class Announcements {
     this.publicURL = publicURL
   }
 
+  get actorUrl (): string {
+    return `${this.publicURL}/v1/${this.mention}/`
+  }
+
+  get outboxUrl (): string {
+    return `${this.actorUrl}outbox`
+  }
+
+  get mention (): string {
+    const url = new URL(this.publicURL)
+    return url.hostname
+  }
+
+  async getActor (): Promise<ActorStore> {
```

Comment:
```suggestion
  getActor (): Promise<ActorStore> {
```

frActor is sync so this should be too